### PR TITLE
Replace bootstrap alert-info/warning <div>s with custom admonitions

### PR DESCRIPTION
**Pull request by mgeier**
This is an experimental work-around feature!

See also #46.

### CODE REVIEW

File: nbsphinx.py [hunk 1]
```diff
@@ -836,6 +836,60 @@ def apply(self):
...
+        flags=re.IGNORECASE)
+    _class_re = re.compile(r"\s*alert\s*alert-(info|warning)\s*")
+    _end_re = re.compile(r"\s*</div\s*>\s*", flags=re.IGNORECASE)
+
+    def apply(self):
+        start_tags = []
+        for node in self.document.traverse(docutils.nodes.raw):
+            if node['format'] != 'html':
+                continue
+            start_match = self._start_re.fullmatch(node.astext())
```
Comment (kmuehlbauer) [inline nbsphinx.py]: fullmatch doesn't work for python < 3.4

**Comment by kmuehlbauer** [comment]
To the fullmatch stuff, I only found this here:

http://stackoverflow.com/questions/30212413/backport-python-3-4s-regular-expression-fullmatch-to-python-2

**Comment by mgeier** [comment]
Thanks for the hint, I've added the suggested work-around using `\Z`.

**Comment by kmuehlbauer** [comment]
I just wanted to say: It work's. Thanks for taking the time @mgeier!

**Pull request merged**